app/controllers/admin_portals_controller.rb

```diff
@@ -34,4 +35,13 @@ def create_user
     @user.save
     redirect_to admins_home_path if @user.save
   end
+
+  def show_pending_users
+    @users = User.all
```

Comment:
I think, to show the pending users, we should get all those with status "pending".

Reply:
Got it.